argo-lite/src/stores/GraphStore.js

```diff
@@ -236,5 +236,10 @@ export default class GraphStore {
     };
     return averageClusteringCoefficient(snapshot);
   }
+
+  @computed
+  get components() {
+    return connectedComponents(this.rawGraph);
```

Comment:
Same as above, use snapshot instead of rawGraph

Reply:
Fixed in a new commit.